jenkins/job-configs/kubernetes-jenkins-pull/charts-pull.yaml

```diff
@@ -0,0 +1,104 @@
+- job:
+    name: 'charts-pull-test-e2e'
+    skip-if-no-test-files: false
+    status-add-test-results: true
+    only-trigger-phrase: false
+
+    concurrent: true
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+        - github:
+            url: 'https://github.com/kubernetes/charts'
+        - throttle:
+            max-total: 0
```

Comment:
do you have the resources to run an unlimited number of these in parallel?

otherwise, you may want to adjust/remove this section and the `concurrent` setting.